testsuite/printf-whole-array/test.osl

```diff
@@ -7,5 +7,9 @@ shader test (
     printf ("iarray = %i\n", iarray);
     printf ("sarray = %s\n", sarray);
     printf ("farray = %f\n", farray);
-}
 
+    // Make sure the elements are reachable
+    printf ("ielems = %d %d %d %d %d\n", iarray[4], iarray[3], iarray[2], iarray[1], iarray[0]);
```

Comment:
I suggest that for the sake of future readers of the test output, you either change this to print the elements in order, or change the printing to say "ielems reversed = %d ..."